sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/integrationtest/BeamSqlBuiltinFunctionsIntegrationTestBase.java

```diff
@@ -153,19 +146,19 @@ public void buildRunAndCheck() {
       PCollection<Row> inputCollection = getTestPCollection();
       System.out.println("SQL:>\n" + getSql());
       try {
-        RowType rowType =
+        Schema schema =
             exps.stream()
-                .map(exp -> RowType.newField(
+                .map(exp -> Schema.Field.of(
```

Comment:
<!--thread_id:cc_178336300_t; commit:79c95678e593da730ba0472b77304ec1f916245e; resolved:0-->
<!--section:context-quote-->
> **akedin** wrote:
> `Schema.newField()` ?

<!--section:body-->
could easily add it, but doesn't seem to save a whole lot?